spec/boat_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'boat'
+
+describe Boat do
+  describe "#initialize" do
+    it "creates a hash with coordinates as keys and false as the default value" do
+      example_boat = Boat.new([[9,1]])
+
+      expect(example_boat.coords_hash[[9,1]]).to eql(false)
+    end
+  end
+
+  describe "#record_hit" do
+    it "sets value of coordinates to true" do
+      example_boat = Boat.new([[9,1]])
+
+      example_boat.record_hit([9,1])
+
+      expect(example_boat.coords_hash[[9,1]]).to eql(true)
```

Comment:
Compare the interface of `example_boat.coords_hash[[9,1]]` with `example_boat.sunk?` - which is more expressive?

Reply:
Sorry I do not understand what you mean here. I have changed 'coords_hash' to 'set_of_coordinates' but I think this is not what you are referring to by this point?